src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -346,4 +347,4 @@
 (def dapp-details-section-body-container
   {:background-color styles/color-white
    :padding-left     10
-   :padding-right    10})
\ No newline at end of file
+   :padding-right    10})
```

Comment:
Out of PR, but you could use `padding-horizontal` here.

Reply:
cc @goranjovic